src/EventStore.Core/ClusterVNodeOptionsExtensions.cs

```diff
@@ -89,6 +89,11 @@ public static class ClusterVNodeOptionsExtensions {
 					ClusterDns = string.Empty
 				}
 			};
+		
+		public static ClusterVNodeOptions WithDevMode(this ClusterVNodeOptions options, bool dev, bool removeDevCerts = false) =>
```

Comment:
Could you add a summary to this extension method please

Reply:
Yup. Done